### DISCUSSION TITLE
BED FRICTION For single point method

### DISCUSSION THREAD

**Discussion by EmreDumlu-UM** [Anura3D/Anura3D_OpenSource#95] [Ideas]
Dear All,

For the purpose of simulation of an open channel flow, I would like to define bed friction downstream. For this reason, I have performed the Sliding Blocks tutorial. However, this tutorial contains solid interactions. If I applied the same method for the liquid, I see Program Exception - access violation error.

Is there a chance to perform bed friction on the downstream contact for the liquid?

Best regards,
Emre

**Comment by murphy-john** [comment]
At this time, there is no way for a pure liquid to interact with a solid (or saturated solid) with the single-point formulation.

The friction for bed loads doesn't occur at a single interface between fluid and soil (see the "law of the wall"). Because the that, the contact formulation used for the solid-solid interface isn't appropriate to use.  

There is ongoing and future research needed to get the functionality to work in the double-point formulation. In MPM, interfaces between materials must be handled very carefully. 

I see you're up in Oxford, MS. Are you at Ole Miss?  I'm at ERDC in Vicksburg. I'd be interested to hear more about your work!
